### PR TITLE
VIH-7871 enable mobile support

### PR DESCRIPTION
**Pull request by AWildBugAppeared**
https://tools.hmcts.net/jira/browse/VIH-7871

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.ts [hunk 1]
```diff
@@ -28,6 +28,14 @@ export class DeviceTypeService {
         );
     }
 
+    isIOS(): boolean {
+        return this.deviceDetectorService.os.toLowerCase() === 'ios';
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.ts:32]: Could use the OS const from node_modules\ngx-device-detector\lib\device-detector.constants.d.ts

**Review by robertbennett1998 — DISMISSED** [review]
LGTM

**Review by robertbennett1998 — APPROVED** [review]
LGTM

File: azure-pipelines.beta.yml [hunk 2]
```diff
@@ -74,19 +74,19 @@ variables:
...
 trigger: none
 pr: none
 schedules:
-- cron: "0 19 * * 0"
-  displayName: Weekly Beta build
-  always: true
-  branches:
-    include:
-    - master
+  - cron: '0 19 * * 0'
```
Comment (FarronF) [inline azure-pipelines.beta.yml:84]: A little concerned about this and the spacing changes in the yml files. Aren't they very spacing sensitive or am I mistaken? I will approve and let the pipelines determine that though.

File: VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.spec.ts [hunk 3]
```diff
@@ -97,6 +97,22 @@ describe('DeviceType', () => {
         expect(service.getOSVersion()).toBe(testOsVersion);
     });
 
+    it('should return whether hte OS is iOS', () => {
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/services/device-type.service.spec.ts:100]: Minor. Misspelling of "the" don't worry about it if there's no other changes to make

File: VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts [hunk 4]
```diff
@@ -5,28 +5,42 @@ import { DeviceTypeService } from '../../services/device-type.service';
...
     ngOnInit() {
-        if (this.deviceTypeService.isDesktop() || this.deviceTypeService.isIpad()) {
-            this.profileService
-                .getUserProfile()
-                .then(profile => this.navigateToHearingList(profile))
-                .catch(error => this.errorService.handleApiError(error));
-        } else {
-            this.router.navigate([pageUrls.UnsupportedDevice]);
-        }
+        this.configService.getClientSettings().subscribe(settings => {
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts:27]: Shouldn't we unsubscribe from this on destroy?

File: VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts [hunk 5]
```diff
@@ -5,28 +5,42 @@ import { DeviceTypeService } from '../../services/device-type.service';
...
+            this.enableAndroidSupport = settings.enable_android_support;
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts:28]: we should just use the value from the settings as its not used anywhere else

File: VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts (hunk 4 above)
Comment (FarronF) [reply VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts:27]: Oh is this an API call that will resolve itself? I would prefer to be more specific here but I'll approve if it is.

**Review by FarronF — APPROVED** [review]

File: azure-pipelines.yml [hunk 6]
```diff
@@ -323,6 +356,35 @@ extends:
       StrykerBreak: 75
     ACTest: ${{ parameters.RunACTests }}
     releaseParameters:
+      variables:
+        - name: EnableAndroidSupport
+          value: false
```
Comment (alberto-hortiguela) [inline azure-pipelines.yml:361]: Why are we setting the values in the yml files instead of using the variables from the group?

**Review by alberto-hortiguela — APPROVED** [review]
LGTM

**Pull request merged**